cli/objects/jira_base.py

```diff
@@ -152,9 +153,21 @@ def create_issue(
 
         return issue
 
-    def search(self, jql_query: str) -> list[str]:
+    def search(self, jql_query: str) -> list[Any]:
         """
-        Performs a Jira JQL query using the Jira connection and returns the results.
+        Performs a Jira JQL query using the Jira connection and returns a list of issues, including all fields.
+
+        Args:
+            jql_query (str): JQL query to run.
+
+        Returns:
+            list[Any]: List of issues that are returned from the query.
+        """
+        return self.connection.search_issues(jql_query, validate_query=True)
+
+    def search_issues(self, jql_query: str) -> list[str]:
+        """
+        Performs a Jira JQL query using the Jira connection and returns a list of strings representing issue keys.
```

Comment:
In line 179 you should call `self.search()`

Reply:
Ack, fixed